components/director/internal/domain/application/resolver.go

```diff
@@ -293,6 +299,12 @@ func (r *Resolver) CreateApplication(ctx context.Context, in graphql.Application
 	return gqlApp, nil
 }
 func (r *Resolver) UpdateApplication(ctx context.Context, id string, in graphql.ApplicationInput) (*graphql.Application, error) {
+	convertedIn := r.appConverter.InputFromGraphQL(in)
+	if len(convertedIn.Name) > applicaionNameMaxLength {
```

Comment:
The validation shouldn't be done here. As you can see the following code, name validation is already done in Service, while ApplicationInput -> Application conversion. Please modify that `Validate()` method which is used.